.changeset/brave-dodos-know.md

```diff
@@ -0,0 +1,5 @@
+---
+'@swisspost/design-system-documentation': patch
+---
+
+Migrate form patterns page from old documentation and add a usage sample to align buttons together in the button component page.
```

Comment:
```suggestion
Migrated form patterns page from old documentation and added a usage sample to align buttons together in the button component page.
```
Maybe no need to talk about the "old documentation", "Documented the form patterns" could be enough.